embulk-standards/src/main/java/org/embulk/standards/CsvTokenizer.java

```diff
@@ -324,6 +326,28 @@ public String nextColumn()
         }
     }
 
+    public String nextColumnOrNull()
+    {
+        String v = nextColumn();
+        if (v == null) {
+            return null;
+        }
+        else if (v.isEmpty() && nullStringOrNull == null) {
```

Comment:
minor: `nullStringOrNull == null` check can be moved to outer `if` block so that following `v.equals(nullStringOrNull)` check is also skipped if `nullStringOrNull` is null.